### PR TITLE
Private fact audit username

### PR DESCRIPTION
**Pull request by garunagiri**
Depending on where x.user.name is being used or is being parsed from, it is now either host.user.name or domain.user.name

### CODE REVIEW

**Comment by garunagiri** [comment]
When x.user.name is being used to access a remote machine, domain.x.x is used. Domain.x.x is also used for Mimikatz cred dump. In other cases, such as running an application as another user on the same local system, host.x.x is used.

File: data/abilities/lateral-movement/c5e75d70-09ae-469f-8717-621a1b7ef946.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     windows:
       psh:
         command: |
-          $username = "#{comp.user.name}";
+          $username = "#{domain.user.name}";
           $password = "#{comp.user.password}";
```
Comment (wbooth) [inline data/abilities/lateral-movement/c5e75d70-09ae-469f-8717-621a1b7ef946.yml]: there should be parity here, if username is domain then so should the password

Comment (garunagiri) [reply data/abilities/lateral-movement/c5e75d70-09ae-469f-8717-621a1b7ef946.yml]: This probably wasn't the best way to do it, but I made a separate branch/PR for each term. So the username and the password do both match, I just made the respective changes across two pull requests. The "Changing comp.user.password to either host.x.x or domain.x.x dependin…" PR has this change

**Comment by privateducky** [comment]
@wbooth got some conflicts here

**Comment by garunagiri** [comment]
@wbooth I've resolved all the conflicts

**Pull request merged**